scripts/jenkins-mender-convert-build.sh

```diff
@@ -0,0 +1,307 @@
+#!/bin/bash
+
+LINARO_COMPILER=gcc-linaro-6.3.1-2017.05-x86_64_arm-linux-gnueabihf
```

Comment:
So far so good, but one must keep in mind that compiler can be changed accordingly to https://tracker.mender.io/browse/MEN-2399.